src/databroker_elasticsearch/converters.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+"""\
+Functions for converting values that are exported to Elasticsearch.
+"""
+
+# handle registration of converter functions ---------------------------------
+
+def register_converter(f, name=None):
+    "Decorator to mark up some function as a converter."
+    nm = f.__name__ if name is None else name
+    _converters[nm] = f
+    return f
+
+_converters = {}
```

Comment:
I would put this as a public dictionary.

Reply:
See https://github.com/NSLS-II/databroker-elasticsearch/pull/11#discussion_r254434078.
The dictionary is for internal implementation of converters registry.